CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ If you send a pull request, please do it against the `master` branch. We are dev
 ***
 
 ## Setup Development Environment
-To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the recommended versions of Node.js (v8) and npm (v5).
+To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the recommended versions of Node.js (v9) and npm (v5).
```

Comment:
Alternatively, we can just avoid having to sync on every change version and just link to the requirements section:

```suggestion
 To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the [required versions of Node.js and npm](https://strapi.io/documentation/3.x.x/getting-started/installation.html#requirements) 
```

What do you think?

Reply:
cc @lauriejim ☝️